commit e1bcb97e29f95edba8a5a7ba5e115efc252e8077
Author:     Mohamed Boudra <boudra.moha@gmail.com>
AuthorDate: Sun Nov 16 13:55:13 2025 +0100
Commit:     Mohamed Boudra <boudra.moha@gmail.com>
CommitDate: Sun Nov 16 13:55:13 2025 +0100

    chore: update app icons and branding assets with black adaptive icon background

diff --git a/packages/app/app.json b/packages/app/app.json
index acb08273b..2620c1ea5 100644
--- a/packages/app/app.json
+++ b/packages/app/app.json
@@ -23,7 +23,7 @@
     },
     "android": {
       "adaptiveIcon": {
-        "backgroundColor": "#E6F4FE",
+        "backgroundColor": "#000000",
         "foregroundImage": "./assets/images/android-icon-foreground.png"
       },
       "edgeToEdgeEnabled": true,
